### ISSUE TITLE
Curl not installed by default?

### ISSUE THREAD

**Issue by brannondorsey** [brannondorsey/ProbeRequestCollectorsKit#67]
`install.sh` relies on curl which in one instance of an ubuntu install I found was missing.

**Comment by JPaulMora** [comment]
add to install script & wiki entry for said curl-less distro

```
CURL_PRESENT=$(which curl) # or whereis curl
if [ -z  $CURL_PRESENT ]
    then
    echo "Please install curl"
else 
 #curl installed!
echo "Looking good!" 
fi
```

**Comment by brannondorsey** [comment]
Thanks for the snippet. I saw that you forked the project :) Were you able to get it running? I don't think this is written anywhere but back when we were actively developing this we were working with the `develop` branch. Also we've got a documentation site here... I should update the repo to reflect that soon: http://probekit.brangerbriz.com.

Cheers!

**Comment by JPaulMora** [comment]
Yes, Most stuff worked out just by cloning. Didn't notice there was a develop branch, check ill check that out. By the way, great work over here!
